test/index.ts

```diff
@@ -396,4 +398,32 @@ describe('retry-axios', () => {
     }
     assert.fail('Expected to throw');
   });
+
+  it('should parse Retry-After header for retry delay if available', async () => {
+    const scopes = [
+      nock(url).get('/').reply(429, undefined, {'Retry-After': '1'}),
+      nock(url).get('/').reply(200, 'toast'),
+    ];
+    interceptorId = rax.attach();
+    const res = await axios({url});
+    assert.strictEqual(res.data, 'toast');
```

Comment:
This isn't actually testing if the delays work.  I know it's annoying, but could I trouble you to set up a test with a sinon timer, and verify it's doing what we expect?  I've had to write similar tests in the past if you're looking for an example:
https://github.com/JustinBeckwith/linkinator/blob/main/test/test.retry.ts#L83

Reply:
Done using the same pattern - also set a higher default retry delay on the rax config to make sure the Retry-After value is actually being used